src/css/lib/custom/jsoneditor-add.scss

```diff
@@ -0,0 +1,6 @@
+@import "../../partials/base";
+
+.monster-jsoneditor {
+	width: 600px;
+	height: 70vh;
+}
```

Comment:
Yeah you're right, I had added these styles since set an initial size to the editor's container is mandatory, but I can just point it out this in the docs and the user can handle it